includes/class-affiliatewp-allow-own-referrals.php

```diff
@@ -0,0 +1,285 @@
+<?php
+/**
+ * Allow Own Referrals Base Class
+ *
+ * @package AffiliateWP\AffiliateWP_Allow_Own_Referrals
+ * @since   1.0.0
```

Comment:
```suggestion
 * @since   1.0.3
```
Update the since annotation for the version the file was introduced.